src/copy.tsx

```diff
@@ -527,11 +526,11 @@ export default {
     sponsors
   },
   about: {
-    title: "Canada's biggest hackathon",
+    title: "About JITHack:",
     sentence1:
-      "Join 1,000+ students from all around the globe at the University of Waterloo to build something incredible in just 36 hours.",
+      "JITHack is a design sprint-like event which takes place on 26.06.2021 which will be held online, the participating team gets 24 hours to build something incredible..",
```

Comment:
@TuzzR Could you change this to **design and development**

Reply:
okay